contributing.md

```diff
@@ -0,0 +1,153 @@
+`pageComponents`|One of `[['featured-cards'], ['content-list']]`|No| Any page that has child pages can list its children in either card or list style at the bottom of the page. Specify the desired style with this key.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+|`featuredChildren`| Array of URLs| No| Similar to `pageComponents`, this displays the children of the current page, but only the selected ones.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|`index`|string|No| If a section landing page needs to be different from the `index.md` file in that directory, this field specifies the corresponding Markdown file name. | 
+
+#### Partials
+
+Partials allow you to reuse snippets of content in multiple places. All partials
+live in the `_partials` top-level directory. To make a new partial, create a new
+`.md` file in this directory. The filename must start with an underscore. Then import it into the target page and reference in the relevant place. See [Formatting examples](_formatting_examples.md).
+
+### Formatting
+
+In addition to all the [regular Markdown formatting][markdown-syntax], the following elements are available for Timescale docs:
+
+- Procedure blocks 
+- Highlight blocks
+- Tabs
+- Custom code blocks
+- Multi-tab code blocks
+- Tags
+
+See [Formatting examples](_formatting_examples.md) for how to use them. 
+
+### Variables
+
+Timescale documentation uses variables for its product names, features, and UI elements in Timescale Console with the following syntax: `$VARIABLE_NAME`. Variables do not work inside the following: 
```

Comment:
```suggestion
Timescale documentation uses variables for its product names, features, and UI elements in  $CONSOLE with the following syntax: `$VARIABLE_NAME`. Variables do not work inside the following: 
```

While I think about it, can use smother this page with variables please.

Reply:
Even though they will not render on github?